ext/sigar/CMakeLists.txt

```diff
@@ -9,28 +9,29 @@ project(sigar)
 
 #--------------------------------------------------------------------
 # Define defintions
-if(WIN32) 
-    ## make sure we only use the smallest set of 
-    ## headers on win32. Otherwise we get clashes 
+if(WIN32)
```

Comment:
Since this is an external dependency I would prefer if we kept the changes to a minimum, i.e. don't push any formatting changes...

Reply:
sigar did not see an update for over 4 years and our CMakeLists already kind of replaced every line from the original. I first thought I should fork sigar, apply my changes and switch the submodule to my repo, but I then found it is actually right in our repo, not a submodule,  and already differs significantly.